lib/tasks/wos.rake

```diff
@@ -0,0 +1,43 @@
+namespace :wos do
+  # @return [Clarivate::LinksClient]
+  def links_client
```

Comment:
defining methods in rake files is an anti-pattern

Reply:
So what do you recommend as the pro-pattern for these simple util methods that make console life a happy dance?

Reply:
Whatever you do recommend, I hope your open to working on changing all the things too, e.g.
```
$ git grep 'def' lib/tasks/
lib/tasks/cap_cutover.rake:  def find_issn_id_identifiers(identifiers_array)
lib/tasks/cap_cutover.rake:  def build_pub_from_sw_and_pubmed(pub)
lib/tasks/cap_cutover.rake:  def build_pub_from_missing_sw_and_pubmed(pub)
lib/tasks/cap_cutover.rake:  def self.convert_manual_publication_row_to_hash(cap_pub_data_for_this_pub, author_id)
lib/tasks/pubmed.rake:  def client
lib/tasks/sul.rake:    method = args[:method] || "rebuild_pub_hash" # default to rebuilding pub_hash, could also rebuild_authorship
lib/tasks/sul.rake:    raise "Method #{method} not defined" unless Publication.new.respond_to? method
lib/tasks/sw.rake:  def harvester
lib/tasks/sw.rake:    args.with_defaults(starting_author_id: 1, ending_author_id: -1)
lib/tasks/sw.rake:    institution = sciencewire_harvester.default_institution
```

Reply:
Yes, I already have work removing 4 of those.

Reply:
Done.